cmd/siac/rentercmd.go

```diff
@@ -452,31 +489,41 @@ func rentercontractscmd() {
 		}
 		w.Flush()
 	}
+
 	if renterAllContracts {
-		if len(rc.OldContracts) == 0 {
+		rc, err := httpClient.RenterExpiredContractsGet()
```

Comment:
need to rename `rc` as this would overwrite the previous api call, and since `RenterExpiredContractsGet()` doesn't return `InactiveContracts` the following `if` statement could potentially say that there are no contracts when the renter has only `InactiveContracts`